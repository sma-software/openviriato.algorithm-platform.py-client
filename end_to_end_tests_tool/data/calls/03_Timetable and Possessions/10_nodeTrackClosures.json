{
    "restResource": "/possessions/node-track-closures",
    "httpMethod": "GET",
    "urlParameters": null,
    "queryParameters": [
        {
            "key": "FromTime",
            "value": "2003-05-01T07%3A30%3A00",
            "optional": false
        },
        {
            "key": "ToTime",
            "value": "2003-05-01T08%3A05%3A45",
            "optional": false
        }
    ],
    "requestBody": null,
    "CSharpMethodName": "GetNodeTrackClosures(timeWindow)",
    "CSharpMethodParameterTypes": [ "SMA.AlgorithmPlatform.AlgorithmInterface.AIDM.ITimeWindow" ],
    "subcategory": "Get Node Track Closures"
}