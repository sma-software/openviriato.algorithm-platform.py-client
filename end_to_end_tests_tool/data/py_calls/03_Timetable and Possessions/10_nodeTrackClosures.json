{
  "PythonParameterFromJson" :
  [
      {
          "ParameterName" : "time_window",
          "aidm_factory": "convert_json_to_time_window",
          "ObjectJson":
          {
              "from_time": "2003-05-01T07:30:00",
              "to_time": "2003-05-01T08:05:45"
          }
      }
  ]
}