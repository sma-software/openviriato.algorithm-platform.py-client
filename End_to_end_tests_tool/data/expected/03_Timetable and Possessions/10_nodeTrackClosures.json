[{"DebugString":"nodetrackclosure:85ZMUS 24","NodeID":621,"NodeTrackID":622,"ClosureTimeWindow":{"FromTime":"2003-05-01T08:00:00","ToTime":"2003-05-02T10:00:00"}},{"DebugString":"nodetrackclosure:85ZMUS 23","NodeID":621,"NodeTrackID":623,"ClosureTimeWindow":{"FromTime":"2003-05-01T08:00:00","ToTime":"2003-05-02T10:00:00"}}]